website/content/v1.5/talos-guides/install/boot-assets.md

```diff
@@ -8,8 +8,132 @@ can be customized further for a specific use case:
 
 * adding [system extensions]({{< relref "../configuration/system-extensions" >}})
 * updating [kernel command line arguments]({{< relref "../../reference/kernel" >}})
+* using custom `META` contents, e.g. for [metal network configuration]({{< relref "../../advanced/metal-network-configuration" >}})
 * generating [SecureBoot]({{< relref "../install/bare-metal-platforms/secureboot" >}}) images signed with a custom key
 
+There are two ways to generate Talos boot assets:
+
+* using [Image Factory]({{< relref "#image-factory" >}}) service (recommended)
+* manually using [imager]({{< relref "#image-factory" >}}) container image (advanced)
```

Comment:
This should be a reference to `imager` docs, right?

Reply:
yes, thanks